extensions/notebook/src/book/bookTrustManager.ts

```diff
@@ -6,11 +6,12 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 import * as constants from './../common/constants';
 import { BookTreeItem } from './bookTreeItem';
-import { BookModel } from './bookModel';
+import { BookModel, BookVersion } from './bookModel';
 
 export interface IBookTrustManager {
 	isNotebookTrustedByDefault(notebookUri: string): boolean;
 	setBookAsTrusted(bookRootPath: string): boolean;
+	setBookAsUnTrusted(bookRootPath: string): boolean;
```

Comment:
Rather we just combine these into a single function `setBookTrusted(bookRootPath: string, isTrusted: boolean)`